plugins/modules/database/saphana/hdbsql.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: hdbsql
+
+short_description: Execute SQL on HANA.
+
+version_added: "1.0.0"
```

Comment:
```suggestion
version_added: "3.2.0"
```
The next minor collection release is 3.2.0. (1.0.0 has been released ~a year ago.)